react/LocationInput.tsx

```diff
@@ -80,8 +111,12 @@ const LocationInput: React.FC<Props> = ({
 
   const handleSubmit: React.EventHandler<
     React.FormEvent<HTMLFormElement> | React.MouseEvent<HTMLButtonElement>
-  > = evt => {
-    evt.preventDefault()
+  > = event => {
+    event.preventDefault()
+    if (invalidPostalCode) {
+      return
+    }
+
     setLoading(true)
     setInvalidPostalCode(false)
```

Comment:
I think this line isn't necessary anymore, right?

Reply:
Since if `invalidPostalCode` is `true`, it will return early. Or am I missing something?

Reply:
Nice catch! Will remove